DEVELOPMENT.md

```diff
@@ -18,7 +18,7 @@ yarn install
 ### 3. Create separate config file for local overrides
 
 ```shell
-touch app-config.local.yaml
+touch docs-config.local.yaml
```

Comment:
just `config.yaml` is not really clear in my opinion. I renamed it to docs-config so it will be more clear in another repos that uses codex.docs (like this https://github.com/nikmel2803/editorjs-docs/blob/main/docs-config.yaml). Just `config` or `app-config` are not clear names in such case.